build.sbt

```diff
@@ -78,7 +91,11 @@ lazy val sharedSettings = MimaSettings.settings ++ scalaVersionSettings ++ Seq(
   // don't use fatal warnings in tests
   scalacOptions in Test ~= (_ filterNot (_ == "-Xfatal-warnings")),
 
-  mimaPreviousArtifacts := Set("org.scalacheck" %% "scalacheck" % "1.14.0"),
+  mimaPreviousArtifacts := {
+    // TODO: re-enable MiMa for 2.13.0-M4 once there is a release out
```

Comment:
Maybe add a ticket to keep track of this?

Reply:
Added a comment https://github.com/rickynils/scalacheck/issues/410#issuecomment-391971774